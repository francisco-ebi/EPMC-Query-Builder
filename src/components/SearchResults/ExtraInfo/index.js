export default function ExtraInfo({
  publication: {
    citedByCount,
    id,
    pmcid
  }
}) {
  return  (
    <p className="extra-info">
      Cited by: {citedByCount} articles |
      PMID: {id} |
      PMCID: {pmcid}
    </p>
  );
}