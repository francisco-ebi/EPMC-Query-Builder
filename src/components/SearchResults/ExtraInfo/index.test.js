import renderer from 'react-test-renderer';
import ExtraInfo from './index';

it('it creates', () => {
  const publication = {
    citedByCount: 11,
    id: 'id',
    pmcid: 'pmcid'
  };
  const component = renderer.create(
    <ExtraInfo publication={publication}/>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});