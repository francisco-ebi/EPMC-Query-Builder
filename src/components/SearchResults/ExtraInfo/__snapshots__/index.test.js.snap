// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it creates 1`] = `
<p
  className="extra-info"
>
  Cited by: 
  11
   articles | PMID: 
  id
   | PMCID: 
  pmcid
</p>
`;
